pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -50,7 +50,13 @@ Before running the following command, ensure that your `dkp` configuration **ref
     For air-gapped:
```

Comment:
This gets a little more weird because of how catalog applications are configured. @takirala I think we'll also need users to update the CLI configuration to either point to the new catalog apps tag or the new repo tar, right?

Reply:
I took a stab at this in https://github.com/mesosphere/dcos-docs-site/pull/4233/commits/76abdae6c7cbcbaf8b79e39d96bfa3657f0ceeb2